flask_stupe/pagination.py

```diff
@@ -35,16 +60,17 @@ def _paginate(cursor, skip=None, limit=None, sort=None, count=True):
         return cursor
 
     def paginate(function_or_cursor=None, skip=None, limit=None, sort=None,
-                 count=True):
+                 count=True, headers=False):
```

Comment:
I'm wondering if we really want to make it an option, or rather to do a breaking change and completely switch to headers.

Reply:
As the current `count` can be misleading (it is in fact the total of matched elements, not the count of returned ones), I'm in favor of this proposition :+1:

Reply:
If we decide to do a breaking change and use headers, then I'd be in favor that we also remove all kind of metadata structure, i.e. switch from this:

```json
{
  "data": {
    "foo": "bar"
  }
}
```

to this:

```json
{
  "foo": "bar"
}
```

and also remove the `status` field for errors.

This would belong to another PR, though, but should be bundled in the same release to avoid multiple breaking change releases.